Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line, and also
 # from the environment for the first two.
-SPHINXOPTS    ?= -n -j "auto" -w "build.log"
+SPHINXOPTS    ?= -n -w "build.log"
```

Comment:
we should keep this

Reply:
SO I get a warning from the sitemap builder with this flag enabled, *but* it doesn't seem to impact the actual contents of the file. Not sure - I am reaching out to the maintainer to understand what the concern is.

Reply:
Without this it is not parallelized and slow.